config/application.rb

```diff
@@ -29,5 +29,12 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
```

Comment:
I'n not super familiar w/  the `rack-cors` gem.  Will this open up requests to anywhere in the app, or is this specific to the `/annotation*` resource?

Reply:
Just to `/annotation*` but from any origin `origins: '*'`

Reply:
Okay, just to rephrase: This only sets `origins: *` under the `/annotation*` path (and does not affect the `origins` header elsewhere (and tbh, I'm not sure we need to care if it does, just a curiosity of the DSL here)?

Reply:
A brief look at https://github.com/cyu/rack-cors/blob/master/lib/rack/cors.rb seems to indicate that `origins` are set for each resource independently.

Reply:
![🆒](https://media.giphy.com/media/XreQmk7ETCak0/giphy.gif)